### PR TITLE
add jsk_spot_startup and update usage documentation

### PR DESCRIPTION
**Pull request by sktometometo**
- add jsk_spot_startup package
- add jsk_spot.rosinstall
- update README.md

Please merge https://github.com/k-okada/jsk_robot/pull/4 before this PR

### CODE REVIEW

File: jsk_spot_robot/jsk_spot_startup/package.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>jsk_spot_startup</name>
+  <version>0.0.0</version>
```
Comment (k-okada) [inline jsk_spot_robot/jsk_spot_startup/package.xml:4]: change to 1.1.0,

add author section

File: jsk_spot_robot/jsk_spot_startup/launch/rviz.launch [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<launch>
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find spot_viz)/rviz/robot.rviz" />
```
Comment (k-okada) [inline jsk_spot_robot/jsk_spot_startup/launch/rviz.launch:2]: should point to `jsk_spot_startup/config/spot.rviz ` ??

File: jsk_spot_robot/jsk_spot_startup/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(jsk_spot_startup)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+)
+
+include_directories(
+)
```
Comment (k-okada) [inline jsk_spot_robot/jsk_spot_startup/CMakeLists.txt:10]: add `install` and `test` section.
```

#############
## Install ##
#############
install(DIRECTORY config launch 
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
   USE_SOURCE_PERMISSIONS
   )


#############
## Testing ##
#############
if(CATKIN_ENABLE_TESTING)
  find_package(catkin REQUIRED COMPONENTS roslaunch roslint)
  file(GLOB LAUNCH_FILES launch/*.launch)
  foreach(LAUNCH_FILE ${LAUNCH_FILES})
    roslaunch_add_file_check(${LAUNCH_FILE})
  endforeach()

  set(ROSLINT_PYTHON_OPTS --max-line-length=180 --ignore=E221,E222,E241) # skip multiple spaces before/after operator
  roslint_python()
  roslint_add_test()
endif()
```

**Comment by k-okada** [comment]
it also nice to have driver and rviz icons. 
https://github.com/jsk-ros-pkg/jsk_robot/pull/1257/files
https://github.com/tork-a/rtmros_nextage/blob/939b0916be2dfea0849284211da63499f39ff708/nextage_moveit_config/NextageROS.desktop.in
you'd better to add `-hold` to xterm option for debugging.

File: jsk_spot_robot/jsk_spot_startup/package.xml (hunk 1 above)
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_startup/package.xml]: update the package.xml
- changed the version to 1.1.0
- changed the maintainer
- added an author section
- added dependencies

File: jsk_spot_robot/jsk_spot_startup/launch/rviz.launch (hunk 2 above)
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_startup/launch/rviz.launch]: Fixed the filepath to rviz config.

**Comment by sktometometo** [comment]
confirmed that shortcuts works well in a real machine.

**Comment by sktometometo** [comment]
@k-okada Please merge this PR if no problem

**Comment by sktometometo** [comment]
This PR is moved to https://github.com/k-okada/jsk_robot/pull/14

**Pull request closed without merging**